content/docs/editing/blocks.md

```diff
@@ -125,6 +125,43 @@ We have defined the structure of our 3 blocks (`content`, `features`, `hero`), a
 
 > Note, since our contentBlock uses the [markdown plugin](/docs/reference/toolkit/fields/markdown/) which is not registered out of the box, we will need to manually import the `react-tinacms-editor` plugin.
 
+## Querying Block Data
+
+Because each item in a list of blocks can have a unique schema, querying this data isn't as straightforward as other types of fields. We'll need use GraphQL's fragment syntax to query the appropriate data shape for each block type.
+
+The fragment names are automatically generated based on the collection name and parent field name. For example, if the collection is `pages`, the field is `blocks`, and the block's name is `hero`, the fragment will be named `PagesBlocksHero`.
```

Comment:
Maybe add something that points to the fact that this maps to querying `__typename`

Reply:
nvm its at the bottom